### PR TITLE
Remove is_minimum_GLSL_version()

### PR DESCRIPTION
**Pull request by asarium**
Since OpenGL Core is required now that function would always return true
so it can be removed.

### CODE REVIEW

File: code/graphics/gropengl.cpp [hunk 1]
```diff
@@ -1463,42 +1438,31 @@ static void init_extensions() {
 	GLSL_version = ver;
 
 	// we require a minimum GLSL version
-	if (!is_minimum_GLSL_version()) {
+	if (GLSL_version < MIN_REQUIRED_GLSL_VERSION) {
```
Comment (Echelon9) [inline code/graphics/gropengl.cpp]: A minor suggestion: you could leave the helper function in, and continue to use it in this `if ()` check. Up to you.

**Comment by Echelon9** [comment]
The name of this PR was a little confusing at first, as on review of the code changes it predominantly removes fixed function rendering code -- rather than the helper function.

I made a small comment within about one suggestion.

File: code/graphics/gropengl.cpp (hunk 1 above)
Comment (asarium) [inline code/graphics/gropengl.cpp]: That would be the only location where that function could be used so I'll keep the current version.

**Comment by asarium** [comment]
Initially all I did was remove that function but then I noticed that there were a couple of empty function so I also removed the remaining fixed function code.

File: code/graphics/gropengl.cpp (hunk 1 above)
Comment (Echelon9) [inline code/graphics/gropengl.cpp]: Fine by me.

**Pull request merged**